modules/core/test/v2/unit/coins/eos.ts

```diff
@@ -78,13 +84,20 @@ describe('EOS:', function () {
     }
   });
 
-  it('isValidMemoId should work', function () {
+  it('isValidMemo should work', function () {
     basecoin.isValidMemo({ value: '1' }).should.equal(true);
     basecoin.isValidMemo({ value: 'uno' }).should.equal(true);
     const string257CharsLong = '4WMNlu0fFU8N94AwukfpfPPQn2Myo80JdmLNF5rgeKAab9XLD93KUQipcT6US0LRwWWIGbUt89fjmdwpg3CBklNi8QIeBI2i8UDJCEuQKYobR5m4ismm1RooTXUnw5OPjmfLuuajYV4e5cS1jpC6hez5X43PZ5SsGaHNYX2YYXY03ir54cWWx5QW5VCPKPKUzfq2UYK5fjAG2Fe3xCUOzqgoR6KaAiuOOnDSyhZygLJyaoJpOXZM9olblNtAW75Ed';
     basecoin.isValidMemo({ value: string257CharsLong }).should.equal(false);
   });
 
+  it('isValidMemoId should work', function () {
+    basecoin.isValidMemoId('1').should.equal(true);
+    basecoin.isValidMemoId('123abc').should.equal(true);
+    const string257CharsLong = '4WMNlu0fFU8N94AwukfpfPPQn2Myo80JdmLNF5rgeKAab9XLD93KUQipcT6US0LRwWWIGbUt89fjmdwpg3CBklNi8QIeBI2i8UDJCEuQKYobR5m4ismm1RooTXUnw5OPjmfLuuajYV4e5cS1jpC6hez5X43PZ5SsGaHNYX2YYXY03ir54cWWx5QW5VCPKPKUzfq2UYK5fjAG2Fe3xCUOzqgoR6KaAiuOOnDSyhZygLJyaoJpOXZM9olblNtAW75Ed';
+    basecoin.isValidMemoId( string257CharsLong).should.equal(false);
```

Comment:
please add `@prettier` annotation to this file and run prettier. You have an extra space after the opening paren here.